src/duration.rs

```diff
@@ -444,9 +458,18 @@ fn div_rem_64(this: i64, other: i64) -> (i64, i64) {
 #[cfg(test)]
 mod tests {
     use super::{Duration, MIN, MAX, OutOfRangeError};
-    use std::{i32, i64};
+
+    #[cfg(feature = "std")]
```

Comment:
Ah sorry what I mean is that testing implies `extern crate test` which implies that you can have `extern crate std` here and then use imports like `ToString` from `std`. That'll avoid any need for `#[cfg]` here.